### PR TITLE
Merge JAX into employee in weekly report

### PR DESCRIPTION
**Pull request by pbchase**
This is a start. We will need to replace JAX in the daily report as well.

### CODE REVIEW

**Comment by pbchase** [comment]
relates to issue #41

**Comment by pbchase** [comment]
@ljwoodley I know this doesn't address the daily report, but it you think it is good WRT the weekly, can you merge this and then take care of a PR for the daily report yourself?

File: weekly_report.Rmd [hunk 1]
```diff
@@ -54,11 +54,24 @@ results_project_read <- redcap_read(redcap_uri = Sys.getenv("URI"),
...
+  mutate(person_type = case_when(
+    person_type == "JAX" ~ "EMPLOYEE",
+    TRUE ~ person_type
+  )) %>%
   filter(!is.na(person_type)) %>% #from Laurence code
   mutate(person_type = str_to_title(person_type))
 
+# replace person_type == JAX
+cases_project_read <- cases_project_read %>%
+    mutate(case_person_type = case_when(
```
Comment (worldwidey) [inline weekly_report.Rmd:70]: Is variable 'case_person_type' should be 'person_type'? 141-144 used 'person_type'.

Comment (ljwoodley) [reply weekly_report.Rmd:70]: @worldwidey `case_person_type` is a variable in the cases REDCap project.

https://redcap-phone.ctsi.ufl.edu/prod/redcap_v10.0.1/Design/data_dictionary_codebook.php?pid=25

**Pull request merged**